File: graphql-dgs-codegen-core/src/integTest/java/com/netflix/graphql/dgs/codegen/cases/dataClassWithFieldHavingNameInCaps/expected/types/SomeType.java
package com.netflix.graphql.dgs.codegen.cases.dataClassWithFieldHavingNameInCaps.expected.types;

import com.fasterxml.jackson.annotation.JsonProperty;
import java.lang.Object;
import java.lang.Override;
import java.lang.String;
import java.util.Objects;

public class SomeType {
  @JsonProperty("CAPSField")
  private String CAPSField;

  public SomeType() {
  }

  public SomeType(String CAPSField) {
    this.CAPSField = CAPSField;
  }

  public String getCAPSField() {
    return CAPSField;
  }

  public void setCAPSField(String CAPSField) {
    this.CAPSField = CAPSField;
  }

  @Override
  public String toString() {
    return "SomeType{CAPSField='" + CAPSField + "'}";
  }

  @Override
  public boolean equals(Object o) {
    if (this == o) return true;
    if (o == null || getClass() != o.getClass()) return false;
    SomeType that = (SomeType) o;
    return Objects.equals(CAPSField, that.CAPSField);
  }

  @Override
  public int hashCode() {
    return Objects.hash(CAPSField);
  }

  public static Builder newBuilder() {
    return new Builder();
  }

  public static class Builder {
    @JsonProperty("CAPSField")
    private String CAPSField;

    public SomeType build() {
      SomeType result = new SomeType();
      result.CAPSField = this.CAPSField;
      return result;
    }

    public Builder CAPSField(String CAPSField) {
      this.CAPSField = CAPSField;
      return this;
    }
  }
}

File: graphql-dgs-codegen-core/src/integTest/kotlin/com/netflix/graphql/dgs/codegen/cases/inputWithDefaultIntValueForArray/expected/types/SomeType.kt
package com.netflix.graphql.dgs.codegen.cases.inputWithDefaultIntValueForArray.expected.types

import com.fasterxml.jackson.`annotation`.JsonProperty
import com.netflix.graphql.dgs.codegen.GraphQLInput
import com.netflix.graphql.dgs.codegen.cases.inputWithDefaultIntValueForArray.expected.Generated
import kotlin.Any
import kotlin.Int
import kotlin.Pair
import kotlin.String
import kotlin.collections.List

@Generated
public data class SomeType(
  @JsonProperty("numbers")
  public val numbers: List<Int?>? = default<SomeType, List<Int?>?>("numbers", listOf(1, 2, 3)),
) : GraphQLInput() {
  override fun fields(): List<Pair<String, Any?>> = listOf("numbers" to numbers)
}
